obspy/io/stationtxt/core.py

```diff
@@ -11,21 +11,24 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from future.builtins import *  # NOQA
-from future.utils import native_str
-from future import standard_library
-with standard_library.hooks():
-    import collections
-
 import csv
 import io
+import re
 import warnings
+import six
 
 import obspy
+from obspy import UTCDateTime
 from obspy.core.inventory import (Inventory, Network, Station, Channel,
                                   Response, Equipment, Site,
                                   InstrumentSensitivity)
 
+from future.builtins import *  # NOQA
+from future.utils import native_str
+from future import standard_library
+with standard_library.hooks():
+    import collections
+
```

Comment:
Oh and please use a plain `import collections`, should work for Python 2.7, I think..